src/sentry/workflow_engine/models/data_source.py

```diff
@@ -29,7 +29,11 @@ class DataSource(DefaultFieldsModel):
     __relocation_scope__ = RelocationScope.Organization
 
     organization = FlexibleForeignKey("sentry.Organization")
+
+    # Should this be a string so we can support UUID / ints?
     query_id = BoundedBigIntegerField()
```

Comment:
In https://github.com/getsentry/sentry/pull/81953, Colleen assigns this enum a string (like `SNUBA_QUERY_SUBSCRIPTION = "SnubaQuerySubscription"`).